regimes/it/categories.go

```diff
@@ -11,8 +11,12 @@ import (
 // Local tax category definitions which are not considered standard.
 const (
 	// https://www.agenziaentrate.gov.it/portale/imposta-sul-reddito-delle-persone-fisiche-irpef-/aliquote-e-calcolo-dell-irpef
-	TaxCategoryIRPEF cbc.Code = "IRPEF"
-	TaxCategoryINPS  cbc.Code = "INPS"
+	TaxCategoryIRPEF    cbc.Code = "IRPEF"
+	TaxCategoryIRES     cbc.Code = "IRES"
+	TaxCategoryINPS     cbc.Code = "INPS"
+	TaxCategoryENASARCO cbc.Code = "ENASARCO"
+	TaxCategoryENPAM    cbc.Code = "ENPAM"
+	TaxCategoryOther    cbc.Code = "OTHER"
```

Comment:
Perhaps for the time being we can simply skip this "other" case and wait for more specific requirements to be defined, it's just simply too easy to abuse and confuse.